### ISSUE TITLE
Deprecate @View in favor of a new class that doesn't conflict with a common android class name

### ISSUE THREAD

**Issue by ghost** [johncarl81/transfuse#57]
I am finding myself qualifying my usage of android.view.View all over the place... would you consider adding a new @TView or @ViewRef or something that doesn't conflict so frequently?

This actually might be an issue with a couple others as well.. I feel like we should avoid name conflicts with common core android components whenever possible.

**Comment by johncarl81** [comment]
@dbachelder , this was you right?  I'm leaning more in this direction now.  I kindof like `@TView`... should we also change `@Activity` to `@TActivity`?

**Comment by dbachelder** [comment]
Yes, please! Would be great to remove ambiguity with any annotation names that conflict with platform classes

**Comment by johncarl81** [comment]
what's your preference on naming convention?  Prepend with a T (Ugh, I hate to copy Android Annotations here, they prepend with E: `@EActivity`)?  Append a 'Ref'?

**Comment by dbachelder** [comment]
hah.. prepending a letter is a tradition as old as Java :) Save everyone the typing and go with `T` prefix.. that's my opinion.
